apis/concierge/authentication/v1alpha1/types_tls.go.tmpl

```diff
@@ -1,11 +1,30 @@
-// Copyright 2020-2022 the Pinniped contributors. All Rights Reserved.
+// Copyright 2020-2024 the Pinniped contributors. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
 package v1alpha1
 
+// CABundleSource provides a source for CA bundle used for client-side TLS verification.
+type CABundleSource struct {
+	// Whether the CA bundle is being sourced from a kubernetes secret or a configmap.
+	// Secrets must be of type kubernetes.io/tls or Opaque.
+	// For configmaps, the value associated with the key is not expected to be base64 encoded.
```

Comment:
Is this sentence potentially confusing? As a user, I don't explicitly base64 encode my cert for a Secret either, right? Should we remove this sentence?

Reply:
will be done in next push